colossalai/inference/config.py

```diff
@@ -393,3 +393,64 @@ class ModelShardInferenceConfig:
     use_cuda_kernel: bool = False
     use_spec_dec: bool = False
     use_flash_attn: bool = False
+
+
+@dataclass
+class GenerationParams:
```

Comment:
trivial: A class name identifying its purpose and usage for diffusion models is preferred.

Reply:
Thanks!